tfx_addons/version.py

```diff
@@ -30,7 +30,7 @@
 # stable release (indicated by `_VERSION_SUFFIX = ''`). Outside the context of a
 # release branch, the current version is by default assumed to be a
 # 'development' version, labeled 'dev'.
-_VERSION_SUFFIX = "dev"
+_VERSION_SUFFIX = ""
```

Comment:
```suggestion
_VERSION_SUFFIX = "rc0"
```

We should publish first release candidate instead of a version